pyne/serpent.py

```diff
@@ -281,11 +319,23 @@ def parse_dep(depfile, write_py=False, make_mats=True):
         else:
             new_filename = depfile.name.rpartition('.')[0] + '.py'
         with open(new_filename, 'w') as pyfile:
-            pyfile.write(f)
+            pyfile.write(full_f)
 
     # Execute the adjusted file
     dep = {}
-    exec(f, dep, dep)
+    try:
+        exec(full_f, dep, dep)
+    except ValueError:
+        correct_footer = form_footer(f, serp2=True)
```

Comment:
maybe add a comment to explain what is attempted if the first `exec` fails

Reply:
looks handled.